node/src/components/event_stream_server.rs

```diff
@@ -16,9 +16,6 @@
 //! This component uses a ring buffer for outbound events providing some robustness against
 //! unintended subscriber disconnects, if a disconnected subscriber re-subscribes before the buffer
 //! has advanced past their last received event.
-//!
-//! For details about the SSE model and a list of supported SSEs, see:
-//! <https://github.com/CasperLabs/ceps/blob/master/text/0009-client-api.md#rpcs>
```

Comment:
I'm not sure about this one, so I'm leaving this comment just to highlight it for you. This link is valid, and the comment is present on dev branch

Reply:
Yeah, we did decide to remove links to CEPS from the rustdocs, so this one isn't an error :)